modules/virt-disabling-dv-gc.adoc

```diff
@@ -5,9 +5,9 @@
 :_content-type: PROCEDURE
 [id="virt-deleting-dvs_{context}"]
 
-= Deleting a data volume
+= Deleting data volumes
 
-You can delete a data volume by using the `oc` command-line interface (CLI).
+You can delete a data volume if needed by using the `oc` command-line interface (CLI), but be aware that doing so also deletes the underlying provisioned volume claim (PVC).
```

Comment:
This content is no longer relevant. The file was supposed to be deleted. I have updated the PR to reflect that. Thanks.

Reply:
Since we give the option to disable the DV garbage collection, then it would be nice to explain how to delete the remaining DV manually and what are the consequences (PVC deletion)

Reply:
@jpeimer We touch on the PVC life cycle in the Note text in the [About data volumes](https://55067--docspreview.netlify.app/openshift-enterprise/latest/virt/virtual_machines/virtual_disks/virt-creating-data-volumes.html#virt-about-datavolumes_virt-creating-data-volumes) module. Are you suggesting adding anything else to the Note?